hyper/contrib.py

```diff
@@ -77,7 +77,8 @@ def get_connection(self, host, port, scheme, cert=None, verify=True,
                 secure=secure,
                 ssl_context=ssl_context,
                 proxy_host=proxy_netloc,
-                proxy_headers=proxy_headers)
+                proxy_headers=proxy_headers,
+                **kwargs)
```

Comment:
We shouldn't be sending arbitrary kwargs from Requests to hyper: there's no reason to assume that those kwargs will match in any sensible way. We should aim to be explicit here.